src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -441,7 +470,8 @@ private bool AddReplicationItemToBatch(ReplicationBatchItem item, OutgoingReplic
             }
 
             // destination already has it
-            if (ChangeVectorUtils.GetConflictStatus(item.ChangeVector, _parent.LastAcceptedChangeVector) == ConflictStatus.AlreadyMerged)
+            if (MissingAttachmentsInLastBatch == false && 
```

Comment:
When will this happen?

Reply:
This is the code that filters items that have lower CV, this will filter the missing attachments if we ignore MissingAttachmentsInLastBatch == false